vm/systemSmartContracts/delegation.go

```diff
@@ -753,24 +805,26 @@ func (d *delegation) unJailNodes(args *vmcommon.ContractCallInput) vmcommon.Retu
 		d.eei.AddReturnMessage("not enough arguments")
 		return vmcommon.FunctionWrongSignature
 	}
-	err := d.eei.UseGas(d.gasCost.MetaChainSystemSCsCost.DelegationOps)
+	//TODO: Shouldn't be checked if there is enough gas for all the nodes which would be unJailed?
+	err := d.eei.UseGas(uint64(len(args.Arguments)) * d.gasCost.MetaChainSystemSCsCost.DelegationOps)
```

Comment:
it is done when calling useGas

Reply:
But this call is missing at all in original branch. We should keep this check added or not?